### ISSUE TITLE
Add a guideline for non-standard features already replaced by standard versions

### ISSUE THREAD

**Issue by queengooborg** [mdn/browser-compat-data#12724]
During my weekly meeting with @foolip, we were talking about ways to reach 100% real values for the API folder.  We came across a couple of categories of features that still have true/null values, but don't seem worthwhile to hunt down the version numbers since they'll be removed already or should be removed.

One of these categories is non-standard, single-engine features, like the [`MozMousePixelScroll` event](https://developer.mozilla.org/en-US/docs/Web/API/Element/MozMousePixelScroll_event).  This is a feature that was only present in Firefox (our data incorrectly states it was in Edge 18, which cannot be right), and even though it still appears to be around in current versions of Firefox, has a standard replacement supported by all major browsers for a long time.  I feel that a feature like this should be considered irrelevant, as it's made obsolete by a standard version.

I propose the following new guideline to consider a feature irrelevant: **a non-standard feature may be considered irrelevant if there is a standard feature implemented in more than one browser engine for over two years.**

CC @ddbeck

**Comment by ddbeck** [comment]
I'm not enthusiastic about this one. I recognize the pain this intended to fix, but I'm not sure this is a good way to fix it. I've got four main concerns about this idea:

1. What happens if the feature is nominally irrelevant but nevertheless has usage? Looking into `MozMousePixelScroll` specifically, I found this bug [which added telemetry earlier this year for removal consideration](https://bugzilla.mozilla.org/show_bug.cgi?id=1241602). I don't know how to interpret telemetry, but it does get _some_ use. It worries me that we would make a determination of irrelevance despite evidence that there are developers for whom it's manifestly not irrelevant.

2. I'm concerned about creating a problem that mirrors the problem we have with all-`false` features. How do we make sure we're removing features that fall under the guideline? How do we make sure such features don't get reintroduced later? How much can a non-standard feature and its standard alternative diverge from each other before they're no longer seen as equivalent? If these questions have complicated answers, is the proposed guideline going to save you any time?

3. Narrowly, I'm concerned that two years is too short a time for things that actually exist. For CSS features in particular, I think it's pretty common practice for web developers to set both standard and non-standard CSS properties for a long time (indeed, there's basically no penalty to do so, with the cascade). I think it's possible that dropping the data (and content) for, say, `-webkit-` prefixed properties and values will make it _harder_ for developers to move off of non-standard features, perhaps defeating the point.

4. Very narrowly, `MozMousePixelScroll` doesn't make a strong case for this proposal. A quick search of Bugzilla turned up [this bug reporting a crash relating to `MozMousePixelScroll`](https://bugzilla.mozilla.org/show_bug.cgi?id=466433), which was reported when Firefox 3 was current and was fixed in Firefox 3.5. It seems to me that the actual solution to this particular case is to get me to review and merge https://github.com/mdn/browser-compat-data/pull/12661.

Again, I'm not unsympathetic. But I'm not convinced that this is the best solution to the problem. I'm not opposed to being convinced, but I am going to need some convincing. If you want to do that, then let's do it, otherwise, I suggest closing this.
